### ISSUE TITLE
[BUG] Problem searching in the Hebrew language

### ISSUE THREAD

**Issue by test01203** [jurialmunkey/skin.arctic.fuse#61]
### Skin section

Other

### Current Behavior

A search in the Hebrew language does not show results
Only after clicking on the name of the movie on the left side
His name is reversed
And then it does work

### Expected Behavior

As soon as we write down the name of the movie, it will find it
If you are unable to solve the problem, then it may be possible for him to reverse the language at the very beginning

### Steps To Reproduce

Search in Kodi
in the search box above

### Screenshots and Additional Info

https://github.com/jurialmunkey/skin.arctic.fuse/assets/104188844/12d4e308-2ec7-4e92-b1d0-9655de8d98fe

In the video I used the movie The Little Mermaid
בת הים הקטנה in hebrew

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
Kodi bug.

Fixed in Kodi v21
